### ISSUE TITLE
Topic classification about XLSR-53，did I do something wrong?

### ISSUE THREAD

**Issue by Rambo06** [pytorch/fairseq#3406]
## ❓ Questions and Help

Hello，
I used XLSR-53 for topic classification on the Chinese dataset and the accuracy rate can reach 80%.
But when I use 1200 hours of Chinese data for self-supervised training after 50 epochs.  The accuracy becomes 50%.
<img width="947" alt="clipboard" src="https://user-images.githubusercontent.com/52806870/112755395-1372c880-9013-11eb-8e07-f3f43b94f3bc.png">

I would like to ask, how many hours of data does self-training XLSR-53 need?
How many epochs need to be trained ？
Did I do something wrong?

I use the following code：
[https://github.com/mailong25/self-supervised-speech-recognition](url)
`python3 pretrain.py --fairseq_path path/to/libs/fairseq --audio_path path/to/audio_directory --init_model path/to/xlsr_53_56k.pt`

Thanks for your answer!
